tests/fuzzing/run-tests.sh

```diff
@@ -16,7 +16,7 @@ git config --global --add safe.directory /crun
 git clean -fdx
 ./autogen.sh
 ./configure --enable-embedded-yajl HFUZZ_CC_UBSAN=1 HFUZZ_CC_ASAN=1 CC=hfuzz-clang CPPFLAGS="-D FUZZER" CFLAGS="-ggdb3 -fsanitize-coverage=trace-pc-guard,trace-cmp,trace-div,indirect-calls"
-make -j "$(nproc)"
+make -j "$(nproc)" check
```

Comment:
this causes the CI to hang.

Could we run `make -j "$(nproc)" && make tests/tests_libcrun_fuzzer`?

Reply:
I was actually just about to commit it.

Reply:
Committed in slightly different form (without `&&`) as there's `set -e` already.